tests/test_api.py

```diff
@@ -203,6 +203,8 @@ def test_export_user(self):
         self.assert_200(response)
         backups = response.json['data']['backups']
         self.assertEquals(len(backups), 1)
+        self.assertTrue('submission_time' in backups[0])
+        self.assertEquals(backups[0]['submission_time'], backups[0]['created'])
```

Comment:
This won't catch bugs where submission_time is always the created time. We should have a test where we change the custom_submission_time attribute

This can probably go in `test_export_final`, which tests submissions